src/app/node-data/basic/provider/openstack/component.ts

```diff
@@ -127,6 +132,8 @@ export class OpenstackBasicNodeDataComponent extends BaseFormValidator implement
       [Controls.CustomDiskSize]: this._builder.control(''),
       [Controls.Image]: this._builder.control('', Validators.required),
       [Controls.AvailabilityZone]: this._builder.control(''),
+      [Controls.InstanceReadyCheckPeriod]: this._builder.control(this._instanceReadyCheckPeriodDefault),
+      [Controls.InstanceReadyCheckTimeout]: this._builder.control(this._instanceReadyCheckTimeoutDefault),
```

Comment:
Just a question: It seems that those fields are not required, but we provide defaults... Would it be "possible" to create it without a specified number or is it wanted that it's set every time?